src/apps/wallet/sign_tx/signing.py

```diff
@@ -183,6 +185,14 @@ class SigningError(ValueError):
     tx_req.details = TxRequestDetailsType()
     tx_req.serialized = None
 
+    decred = coin.curve_name == 'secp256k1-decred' # TODO: maybe improve this detection method
```

Comment:
I pushed the change to master where you can just use `coin.decred`, so drop this line completely.